### PR TITLE
SCB-1447 add samples

### PR DESCRIPTION
**Pull request by kakulisen**
add samples module and add sample for toolkit-maven-plugin

Signed-off-by: kakulisen <18813972746@163.com>

### CODE REVIEW

**Comment by MabinGo** [comment]
Thanks @kakulisen , I think maybe we can provide more simpler example. For other existing examples, we can use them directly and no need to submit code to the toolkit project.

**Comment by kakulisen** [comment]
Thanks, I think this is a good idea. I will improve it in the next commit

**Comment by MabinGo** [comment]
@kakulisen any news?

**Comment by kakulisen** [comment]
yes, there is still some new content to commit later

**Comment by kakulisen** [comment]
All content has been committed, please help to check, thanks

File: samples/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# samples
+
+It's the samples of ServiceComb Toolkit
+1. toolkit-maven-plugin-sample
```
Comment (MabinGo) [inline samples/README.md]: Content and implementation do not match,has the description been written yet?

File: samples/generate-from-code-sample/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+## Prerequisites
+You will need:
+1. [JDK 1.8][jdk]
```
Comment (MabinGo) [inline samples/generate-from-code-sample/README.md]: recommend to add the links both of JDK and Maven

File: samples/generate-from-code-sample/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+2. [Maven 3.x][maven]
+
```
Comment (MabinGo) [inline samples/generate-from-code-sample/README.md]: recommend to add the description of configuration

File: samples/generate-from-code-sample/pom.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.servicecomb.toolkit</groupId>
+        <artifactId>toolkit-maven-plugin</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+        <configuration>
+          <!-- 输入源。设置为 code，表示解析当前代码；设置为 contract，表示解析指定目录的契约文件。不设置则默认为 code -->
```
Comment (MabinGo) [inline samples/generate-from-code-sample/pom.xml]: recommend to comment in English

File: samples/generate-from-contract-sample/README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+## Prerequisites
+You will need:
+1. [JDK 1.8][jdk]
```
Comment (MabinGo) [inline samples/generate-from-contract-sample/README.md]: recommend to add the links both of JDK and Maven

File: samples/generate-from-contract-sample/README.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+2. [Maven 3.x][maven]
+
```
Comment (MabinGo) [inline samples/generate-from-contract-sample/README.md]: recommend to add the description of configuration

File: samples/generate-from-contract-sample/pom.xml [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  <artifactId>generate-from-contract-sample</artifactId>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.servicecomb.toolkit</groupId>
+        <artifactId>toolkit-maven-plugin</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+        <configuration>
+          <!-- 输入源。设置为 code，表示解析当前代码；设置为 contract，表示解析指定目录的契约文件。不设置则默认为 code -->
```
Comment (MabinGo) [inline samples/generate-from-contract-sample/pom.xml]: recommend to comment in English

File: samples/verify-with-code-sample/README.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
+## Prerequisites
+You will need:
+1. [JDK 1.8][jdk]
```
Comment (MabinGo) [inline samples/verify-with-code-sample/README.md]: recommend to add the links both of JDK and Maven

File: samples/verify-with-code-sample/README.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+2. [Maven 3.x][maven]
+
```
Comment (MabinGo) [inline samples/verify-with-code-sample/README.md]: recommend to add description of configuration

File: samples/verify-with-code-sample/pom.xml [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+  </modules>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.servicecomb.toolkit</groupId>
+        <artifactId>toolkit-maven-plugin</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+        <configuration>
+          <!-- 输入源。设置为 code，表示解析当前代码；设置为 contract，表示解析指定目录的契约文件。不设置则默认为 code -->
```
Comment (MabinGo) [inline samples/verify-with-code-sample/pom.xml]: It's better to comment in  English

File: samples/generate-from-code-sample/README.md (hunk 3 above)
Comment (kakulisen) [reply samples/generate-from-code-sample/README.md]: ok, I will the description of configuration in the next commit.

File: samples/generate-from-code-sample/README.md (hunk 2 above)
Comment (kakulisen) [reply samples/generate-from-code-sample/README.md]: ok, I will add the links both of them in the next commit.

File: samples/README.md (hunk 1 above)
Comment (kakulisen) [reply samples/README.md]: Thanks for your reminder, I forgot to update the README.md and the content has expired. I will update it in the next commit.

File: samples/generate-from-code-sample/pom.xml (hunk 4 above)
Comment (kakulisen) [reply samples/generate-from-code-sample/pom.xml]: ok, I will translate it

**Comment by MabinGo** [comment]
LGTM

**Pull request merged**